scripts/helper.py

```diff
@@ -8,6 +8,19 @@
 _htid_decode_table = str.maketrans(_htid_decode_map)
 
 
+def get_stub_dir(source, vol_id):
+    """
+    Returns the stub directory for the specified volume (vol_id) and
+    source type (source)
+    """
+    if source == "Gale":
+        return vol_id[::3][1:]
+    elif source == "HathiTrust":
+        return vol_id.split(".", maxsplit=1)[0]
```

Comment:
would be helpful to have a one line comment or example for each of these

logic looks good

Reply:
How's it look now?

Reply:
looks great!